docs/gitlab_integration.md

```diff
@@ -0,0 +1,132 @@
+# 1. Install/register GitLab Runner
+GitLab Runner is the open source project that is used to run your jobs and
+send the results back to GitLab. You have to
+[install](https://docs.gitlab.com/runner/install/) it on a machine where
+your analysis will be performed.
+
+Once GitLab Runner is installed, you need to register it with GitLab.
+You can use the following command:
+```sh
+sudo gitlab-runner register \
+    --non-interactive \
+    --url "https://mycompany.gitlab.com" \
+    --registration-token "$REGISTRATION_TOKEN" \
+    --description "codechecker" \
+    --executor "shell"
+```
+
+The token can be found in your projects settings under
+`settings/ci_cd/Set up a specific Runner manually`. You can find more
+information how to
+[register a GitLab Runner](https://docs.gitlab.com/runner/register/index.html).
```

Comment:
Shouldn't this link point to this documentation https://docs.gitlab.com/ee/ci/runners/ ? I found most of description on where to generate the token there.